base/linalg/symmetric.jl

```diff
@@ -138,7 +140,7 @@ A_mul_B!{T<:BlasComplex,S<:StridedMatrix}(C::StridedMatrix{T}, A::StridedMatrix{
 *(A::HermOrSym, B::HermOrSym) = full(A)*full(B)
 *(A::StridedMatrix, B::HermOrSym) = A*full(B)
 
-bkfact(A::HermOrSym) = bkfact(A.data, symbol(A.uplo), issymmetric(A))
+bkfact(A::HermOrSym) = bkfact(full(A.data), symbol(A.uplo), issymmetric(A))
```

Comment:
I can't point to any specific literature but searching for "sparse", "indefinite", and "pivoting" would probably give you something. @tkelman is more into this so he might be able to help.